static/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html data-bs-theme="dark" lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta content="width=device-width, initial-scale=1" name="viewport">
+		<title>ankerctl</title>
+		<link crossorigin="anonymous" href="//cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" rel="stylesheet">
+	</head>
+	<body>
+		<div class="container my-5">
+			<div class="p-2 text-center bg-body-tertiary rounded-3">
+				<h1 class="text-body-emphasis display-3">ankerctl</h1>
+				<p class="lead">Congratulations on running ankerctl&hellip;</p>
+			</div>
+		</div>
+		<div class="container mb-4">
+			<div class="row row-cols-1 row-cols-md-1 row-cols-lg-2 row-cols-xxl-2 g-3 p-2">
+				<div class="col-12 col-lg-12">
+					<h2 class="display-6 fs-2 mb-2">Connecting PrusaSlicer/SuperSlicer</h2>
+					<ol class="list-group list-group-numbered">
+						<li class="list-group-item">Go to "Printer settings" tab</li>
+						<li class="list-group-item">Click "Gear"⚙️ (Add/Edit Physical Printer)</li>
+						<li class="list-group-item">Fill in "Descriptive name for the printer" with whatever you like</li>
+						<li class="list-group-item">Select your Ankermake printer profile from the dropdown menu</li>
+						<li class="list-group-item">Under "Host type" select "OctoPrint"</li>
+						<li class="list-group-item">In "Hostname, IP or URL" fill in with: <kbd>{{ configHost }}:{{ configPort }}</kbd><a class="text-decoration-none" href="#" id="configData">📋</a></li>
+						<li class="list-group-item">Leave "API Key / Password" blank</a></li>
+						<li class="list-group-item">Click "Test" to validate input</li>
+						<li class="list-group-item">Click "Ok"</li>
+						<li class="list-group-item">Go and slice yor files</li>
+						<li class="list-group-item">Press "G&#x3E;" button to start direct upload</li>
+						<li class="list-group-item">Press "Upload and print" - the "Upload"-only is not supported</li>
+						<li class="list-group-item">Enjoy printing with ankerctl 😉</li>
+					</ol>
+				</div>
+				<div class="col-12 col-lg-12 text-center">
+					<img src="static/prusaslicer-setup.png" class="img-thumbnail" alt="step 1">
+					<img src="static/prusaslicer-setup-2.png" class="img-thumbnail mt-2" alt="step 2">
+					<img src="static/prusaslicer-setup-3.png" class="img-thumbnail mt-2" alt="step 3">
+					<img src="static/prusaslicer-setup-4.png" class="img-thumbnail mt-2" alt="step 4">
+				</div>
+			</div>
+			{{ configData }}
+		</div>
+		<script crossorigin="anonymous" integrity="sha384-ENjdO4Dr2bkBIFxQpeoTz1HIcje39Wm4jDKdf19U8gI4ddQ3GYNS7NTKfAdVQSZe" src="//cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/js/bootstrap.bundle.min.js"></script>
```

Comment:
For most sites, I'm a big fan of CDNs, but here I'm not so sure.

It's not unreasonable for people to want to use this project where there is little or no internet connectivity.

How much space would it take to vendor a copy of these in, say, `/static/vendor/...` ?